docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -435,6 +435,20 @@ Events of pull requests.
 
 ### Domain 4 - CI/CD(WIP)
 
+#### cicd_scopes
+
+Information about Jenkins Job, GitHub Action or Gitlab CI. 
+
+| **field**      | **type** | **length** | **description**                                                                                                                                                                                                | **key**        |
+| :------------- | :------- | :--------- |:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| :------------- |
+| `id`           | varchar  | 255        | A cicd_scope's `id` is composed of "< plugin >:< Entity >:< PK0 >[:PK1]..."<br/>For example, a Github cicd_scope's id is like "< github >:< GithubRepos >< GithubRepoId >". Eg. 'github:GithubRepos:384111310' | PK             |
+| `name`         | varchar  | 255        | The name of cicd_scope.                                                                                                                                                                                        |                |
+| `description`  | varchar  | 255        | The description of cicd_scope.                                                                                                                                                                                 |                |
+| `url`          | varchar  | 255        | The url of cicd_scope. Eg. https://github.com/apache/incubator-devlake or https://jenkins.xxx.cn/view/PROD/job/OPS_releasev2/                                                                                  |                |
+| `created_date` | datetime | 3          | cicd_scope creation date                                                                                                                                                                                       |                |
+| `updated_date` | datetime | 3          | Last full update was done for this cicd_scope                                                                                                                                                                  |                |

```

Comment:
"Last full update was done for this cicd_scope" -> "Date of the last data collection for this cicd_scope"

Is emphasizing "full update" important here?

Reply:
`Date of the last data collection for this cicd_scope` is better, hh